<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::BookmarksImporterWidget</class>
 <widget class="QWidget" name="Otter::BookmarksImporterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>131</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="removeExistingCheckBox">
     <property name="text">
      <string>Remove existing bookmarks</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="removeDependentStackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="removePage">
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="2">
        <layout class="QFormLayout" name="removeForm">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="folderLabel">
           <property name="text">
            <string>Import into folder:</string>
           </property>
           <property name="buddy">
            <cstring>folderComboBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="folderLayout">
           <item>
            <widget class="Otter::BookmarksComboBoxWidget" name="folderComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="newFolderButton">
             <property name="text">
              <string>Newâ€¦</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="allowDuplicatesCheckBox">
         <property name="text">
          <string>Allow to duplicate already existing bookmarks</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="noRemovePage">
      <layout class="QFormLayout" name="formLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QCheckBox" name="importToSubfolderCheckBox">
         <property name="text">
          <string>Import into subfolder</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QFrame" name="noRemoveFrame">
         <layout class="QFormLayout" name="noRemoveBox">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="1">
           <widget class="Otter::LineEditWidget" name="subfolderNameLineEditWidget">
            <property name="toolTip">
             <string>Leave empty to import into main folder</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="subfolderNameLabel">
            <property name="text">
             <string>Subfolder name:</string>
            </property>
            <property name="buddy">
             <cstring>subfolderNameLineEditWidget</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Otter::BookmarksComboBoxWidget</class>
   <extends>QComboBox</extends>
   <header>src/ui/BookmarksComboBoxWidget.h</header>
  </customwidget>
  <customwidget>
   <class>Otter::LineEditWidget</class>
   <extends>QLineEdit</extends>
   <header>src/ui/LineEditWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
